src/Console/Commands/EventStoreReplay.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace DigitalRisks\LaravelEventStore\Console\Commands;
+
+use DigitalRisks\LaravelEventStore\Client;
+use DigitalRisks\LaravelEventStore\Contracts\CouldBeReceived;
+use DigitalRisks\LaravelEventStore\EventStore as LaravelEventStore;
+use Illuminate\Console\Command;
+use ReflectionClass;
+use ReflectionProperty;
+use Illuminate\Support\Facades\Event;
+use Rxnet\EventStore\Data\EventRecord as EventData;
+use Rxnet\EventStore\Record\JsonEventRecord;
+
+class EventStoreReplay extends Command
+{
+    protected $signature = 'eventstore:replay {stream} {events}';
+
+    protected $description = 'Replay a single or range of events';
+    protected $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        $stream = $this->argument('stream');
+        $events = explode('-', $this->argument('events'));
+
+        if (count($events) > 1) {
+            $events = range($events[0], $events[1]);
+        }
+
```

Comment:
🧗 It doesn't handle case we I pass `foo-bar`. Please throw exception if the range is invalid - `filter_validate_int` should be helpful.

Separate issue is - when we define 0-1000000 on a stream where there are not enough events. This is not handled as well.

Reply:
>Separate issue is - when we define 0-1000000 on a stream where there are not enough events. This is not handled as well.

Just thinking from a usage perspective, this would be called with a very specific event or range in mind. I wanted to prevent checking if the first and last event exist and then going back and retrieving the events again. It felt like clutter to me. Would you be happy with an exception in the foreach if the response statusCode !== 200?

Reply:
Yeah - exception would be good for first event that doesn't exist. You are still checking if event exists as that's what ` $this->client->get("/streams/$stream/$event"` is doing. Simply handling it somehow when 404 will be enough

Reply:
Added test for 404, but Guzzle already throws an exception so it is actually handled and breaks the foreach.